Library/Formula/bash-completion.rb

```diff
@@ -32,8 +33,15 @@ def install
 
     system "./configure", "--prefix=#{prefix}"
     system "make", "install"
+  end
 
-    unless (compdir/"brew_bash_completion.sh").exist?
+  def post_install
+    # This symlink needs to be created in the post-install step. Otherwise, a
+    # copy of our official bash completion for `brew` will end up in the bottle,
+    # which is not very helpful in keeping this up-to-date as Homebrew updates.
+    brew_comp = compdir/"brew_bash_completion.sh"
+    unless brew_comp.symlink?
```

Comment:
Not strongly. However, since the checksums of all prior versions are known, I wouldn't want to be the one explaining to a user why I deleted his work, though that is unlikely to occur. Perhaps for "repair requires delete" there should be some kind of lost-and-found to which the file is moved rather than deleted.